resources/install/debian/postinst

```diff
@@ -96,8 +96,23 @@ case "$1" in
         # and this breaks startup script
         sed -i 's/\$JVB_EXTRA_JVM_PARAMS//g' $CONFIG
 
-        if ! grep -q "org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP" "$NEW_JITSI_CONFIG" ;then
-            echo "org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP=focus@auth.${JVB_HOSTNAME}/.*" >> $NEW_JITSI_CONFIG
+        if ! grep -q "#org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP" "$NEW_JITSI_CONFIG" ;then
+            sed -i 's/org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP/#org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP/g' $NEW_JITSI_CONFIG
+        fi
+
+        if ! grep -q "org.jitsi.videobridge.xmpp.user.shard.MUC_JIDS" "$NEW_JITSI_CONFIG" ;then
+            echo "org.jitsi.videobridge.ENABLE_STATISTICS=true" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.STATISTICS_TRANSPORT=muc" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.xmpp.user.shard.HOSTNAME=localhost" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.xmpp.user.shard.DOMAIN=auth.$JVB_HOSTNAME" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.xmpp.user.shard.USERNAME=jvb" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.xmpp.user.shard.PASSWORD=$JVB_SECRET" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.xmpp.user.shard.MUC_JIDS=JvbBrewery@internal.auth.$JVB_HOSTNAME" >> $NEW_JITSI_CONFIG
+            echo "org.jitsi.videobridge.xmpp.user.shard.MUC_NICKNAME=$(uuidgen)" >> $NEW_JITSI_CONFIG
+        fi
+
+        if ! grep -q -e "--apis=," "$CONFIG" ;then
```

Comment:
Does this disable REST if it has been enabled manually?

Reply:
And also remove other `JVB_OPTS` that have been added

Reply:
oh, maybe you are right, that can break stuff ... I just need to disable the component if enabled ... I need to re-think it ... We already merged it for jvb1.0 ... probably needs an update and there 
So it can be any of those then:
```
--apis=xmpp
--apis=rest
--apis=xmpp,rest
--apis=rest,xmpp
```
And it must become `--apis=,` or `--apis=rest`.

Reply:
@bgrozev WDYT about this:
```
APIS_VALUE=`grep -oPe '--apis=\K(\w*,*\w*)' $CONFIG`
if [[ $APIS_VALUE == *"xmpp"* ]]; then
    APIS_NEW_VALUE=${APIS_VALUE/xmpp/}
    if [ -z "$APIS_NEW_VALUE" ]; then
        APIS_NEW_VALUE=","
    fi
    sed -i 's/--apis=$APIS_VALUE/--apis=$APIS_NEW_VALUE/g' $CONFIG
fi
```
That produces as a result:
```
--apis=,
--apis=rest
--apis=,rest
--apis=rest,
```